### ISSUE TITLE
Wrong summary in php8

### ISSUE THREAD

**Issue by koekaverna** [tideways/php-xhprof-extension#107]
I tried extension with php8 and there is a problem in summary it shows 0 values: 
![xhgui](https://user-images.githubusercontent.com/20415858/128884312-4182e60f-8e72-4a45-a48c-b9f186ac4c0b.png)
There is two request to the same page, first request is php8 and second is php7.4

**Comment by paulchyrilov** [comment]
PHP Version 8.0.8
tideways_xhprof
Version | 5.0.4
Clock Source | clock_gettime

'main()' function has no relation to the child, like it was on 7.x 'main()==>'
![Clipboard - 25 мая 2022 г , 09_32](https://user-images.githubusercontent.com/1749200/170211232-a511d181-8be3-4c4e-975d-b867edeef2c4.png)

it seems this causes wrong numbers (looks like some default numbers)for main() during saving profile

on php7.4 it was 
![image](https://user-images.githubusercontent.com/1749200/170214387-d956b183-4a66-47ac-b274-91b2512b1f3d.png)


And in the end this causes wrong displaying of profiles list and wrong calculation of percentages inside profile view

**Comment by stronk7** [comment]
Any insight here? It seems that no changes related to this have happened since 2020, and with php74 becoming EOL... we are running out of options if this doesn't calculate totals properly under php80/81.

Ciao :-)

**Comment by hotrush** [comment]
@stronk7 try `pecl install xhprof`, it works for me

**Comment by stronk7** [comment]
> @stronk7 try `pecl install xhprof`, it works for me

Yeah, thanks, @hotrush . That's the PECL (https://github.com/longxinH/xhprof , fork of the old/original, recently "revived"). As far as we support both flavours... I just was asking if this is going to be updated to support newer PHP versions or no, just that.

Ciao :-)
